<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SubtitlesFixerMainWindow</class>
 <widget class="QMainWindow" name="SubtitlesFixerMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>569</width>
    <height>391</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Subtitles fixer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QPushButton" name="pushButtonSelectFoulder">
        <property name="text">
         <string>Select foulder</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEditFoulderPath"/>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QGroupBox" name="groupBoxProperties">
        <property name="title">
         <string>Parameters</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="labelIncreaseFontSize">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Increase font size value:</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="2">
          <widget class="QFontComboBox" name="fontComboBox"/>
         </item>
         <item row="0" column="2">
          <widget class="QSpinBox" name="spinBoxIncreaseSize">
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimum">
            <number>-99</number>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QCheckBox" name="checkBoxFontsize">
           <property name="text">
            <string>Enabled</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="labelUpdateFontName">
           <property name="text">
            <string>Update font name:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QCheckBox" name="checkBoxFontName">
           <property name="text">
            <string>Enabled</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="labelCodec">
           <property name="text">
            <string>Codec for save:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QComboBox" name="comboBoxCodecs">
           <item>
            <property name="text">
             <string>UTF-8</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Windows-1251</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QCheckBox" name="checkBoxBomUsage">
           <property name="text">
            <string>With BOM</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QPushButton" name="pushButtonStart">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Start fix</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QTextEdit" name="textEdit"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
